windows/README.md

```diff
@@ -91,21 +91,63 @@ pip install -r requirements.txt
 ./scripts/gendriver.sh /path/to/Windows/ISOs
 ```
 
-The ```gendriver.sh``` script takes as input a folder containing the ISOs of all Windows versions to support.
+The `gendriver.sh` script takes as input a folder containing the ISOs of all Windows versions to support.
 It extracts all kernel binaries from the ISOs, downloads corresponding symbol files (PDB), then generates
-the ```driver/src/winmonitor_gen.c``` file.
+the `driver/src/winmonitor_gen.c` file.
 
 Finally, rebuild the driver. Note that new OS versions may considerably change the kernel structure, requiring upgrading
 the generation scripts.
 
+Code Coverage for Windows Binaries
+==================================
+
+Binaries produced by Microsoft tools contain line information in PDB files. These files have a proprietary
+format and are not readable by Linux tools. This poses several challenges in order to covert program counters
+produced by S2E's coverage plugin into say LCOV reports.
+
+The following instructions explain how to get LCOV coverage reports for Windows binaries.
+
+- Build the solution in Visual Studio
+
+- Run `pdbparser.exe` as follows:
+
+    ```
+    ./x64/Release/pdbparser.exe -l my_binary.exe my_binary.pdb > my_binary.exe.lines
+    ```
+
+    The `*.lines` file contains line information in JSON format.
+
+- Run your binary in S2E using `s2e-env`. At the end of the run, you should have `tb-coverage*.json`
+  files in your `s2e-last` folder.
+
+- Run `s2e coverage lcov my_binary`. Make sure that the `my_binary.exe.lines` is located in the same
+  directory as `my_binary.exe`.  This should produce the `coverage.info` file, which contains
+  LCOV coverage info.
+
+- You must use LCOV on Windows in order to generate the report, because the LCOV files contain Windows path.
+  You can also patch the files yourself to convert the paths inside to the Linux format. Run the following command in MSYS after
+  installing LCOV from [here](https://github.com/linux-test-project/lcov`):
```

Comment:
The link is broken, it tries to go to https://github.com/linux-test-project/lcov%60

There's a spurious back-tick in there!

Reply:
thanks!